Source/WebKit/WebProcess/GPU/webrtc/MediaRecorderPrivate.cpp

```diff
@@ -89,7 +89,7 @@ MediaRecorderPrivate::~MediaRecorderPrivate()
     m_connection->send(Messages::RemoteMediaRecorderManager::ReleaseRecorder { m_identifier }, 0);
 }
 
-void MediaRecorderPrivate::videoFrameAvailable(VideoFrame& videoFrame, VideoFrameTimeMetadata)
+void MediaRecorderPrivate::videoFrameAvailable(WebCore::VideoFrame& videoFrame, VideoFrameTimeMetadata)
```

Comment:
Not needed, WebCore is already in scope.

Reply:
I had to add this because of `using namespace PAL` at the top of the file results in a name clash